govstack_test_harness_api/building_blocks/bb-digital-registries/services/services.py

```diff
@@ -0,0 +1,96 @@
+import os
+from types import SimpleNamespace
+from django.contrib.auth.models import AnonymousUser
+from graphene import Schema
+from graphene.test import Client
+from unittest import mock
+from core.schema import Query as CoreQuery, Mutation as CoreMutation
+from django.apps import apps
+
+app_config = apps.get_app_config('govstack_test_harness_api')
+
+
+def get_client(schema, query, mutation):
+    return Client(schema=schema(query=query, mutation=mutation))
+
+
+def create_base_context():
+    user = mock.Mock(is_anonymous=False)
+    user.has_perm = mock.MagicMock(return_value=False)
+    return SimpleNamespace(user=user)
+
+
+def get_context(request):
+    if request.user.is_authenticated:
+        context = create_base_context()
+        context.user = request.user
+    else:
+        context = SimpleNamespace(user=request.user)
+    return context
+
+
+def get_query_content_values(query_content: dict, registry_name: str, version: str) -> dict:
+    content_values = {}
+    if not query_content:
+        return {}
+
+    json_mapping = app_config.digital_registry.get(registry_name, {}).get(version, {})
```

Comment:
Here, you're explicitly using the digital_registry configuration. However, the name of the function, `get_query_content_values`, suggests a more generic purpose. It might be better to pass `app_config.digital_registry` as an input to the function and handle it accordingly. Alternatively, you could pass the entire mapping as an input, since the specific line `app_config.digital_registry.get(registry_name, {}).get(version, {})` is only applicable to the structure of the digital registry configuration. Therefore, there's no need to define it inside this function.

Also I don't think that consecutive gets in fetching config is a good solution. I think it'd be better to fail the process if the registry_name or version is not handled rather than return empty data. It could lead developers to think that their unhandled input is valid and there are no entries which is not true. You can raise custom exception in here and in view add general exception handler and return BadRequest status.